cql3/statements/ks_prop_defs.cc

```diff
@@ -114,13 +108,43 @@ data_dictionary::storage_options ks_prop_defs::get_storage_options() const {
     return opts;
 }
 
+std::optional<unsigned> ks_prop_defs::get_initial_tablets(const sstring& strategy_class) const {
+    // FIXME -- this should be ignored somehow else
```

Comment:
The "problem" is that if you add `TABLETS = { ... }` when creating a ks with, say, SimpleStrategy it will won't cause any errors, replication strategy will remain vnode-based, _but_ `system_schema.scylla_keyspaces` will be updated with `initial_tablets` value which doesn't look correct. So there are two options here

1. bail out with error if `CREATE KEYSPACE` asks for `TABLETS` _AND_ for `REPLICATION` with class that doesn't support tablets
2. ignore `TABLETS` from `CREATE KEYSPACE` in case `REPLICATION`'s class doesn't support tablets

I picked the latter. WDYT?